coordinator/src/main/scala/com/socrata/datacoordinator/secondary/SecondaryWatcher.scala

```diff
@@ -26,7 +26,7 @@ class SecondaryWatcher[CT, CV](universe: => Managed[SecondaryWatcher.UniverseTyp
                                claimantId: UUID,
                                claimTimeout: FiniteDuration,
                                timingReport: TimingReport) {
-  import SecondaryWatcher.log
+  import SecondaryWatcher._
```

Comment:
Switching `SecndaryWatcher.log` to `SecondaryWatcher._` seems weird here. Are you pulling in a lot of things from there?